addon/templates/components/bs4/bs-form/element/errors.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if show}}
+  <div class={{feedbackClass}}>{{messages.firstObject}}</div>
+{{/if}}
```

Comment:
The only difference is the element tag. BS4 uses div tags to render validation messages, while BS3 uses span tags. As far as i have seen, it doesn't seem to affect the styling in any way. I changed it to keep it closer to the BS4 standard. But i guess for keeping maintainability we could keep the errors template in common and ignore the tag difference. What is your opinion on this @simonihmig ?